src/AbstractFixer.php

```diff
@@ -89,21 +89,13 @@ public function supports(\SplFileInfo $file)
         return true;
     }
 
-    /**
-     * {@inheritdoc}
-     */
     public function configure(array $configuration = null)
     {
         if (!$this instanceof ConfigurationDefinitionFixerInterface) {
-            throw new \LogicException('Cannot run method for class not implementing `ConfigurationDefinitionFixerInterface`.');
+            throw new \LogicException(sprintf('Cannot configure using Abstact parent, child not implementing "%s".', ConfigurationDefinitionFixerInterface::class));
```

Comment:
I think this new message is a bit confused, but I agree the previous one was worse. By the way, using `::class` won't work on PHP lower than 5.5.